src/main/java/com/internship/socialnetwork/controller/PostController.java

```diff
@@ -26,32 +27,58 @@ public class PostController {
 
     private final PostService postService;
 
+    private final CommentService commentService;
+
     @GetMapping
     public ResponseEntity<List<PostDto>> getAll() {
         return ResponseEntity.ok(postService.getAll());
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<PostDto> getOne(@RequestParam Long id) {
+    public ResponseEntity<PostDto> getOnePost(@PathVariable Long id) {
         return ResponseEntity.ok(postService.get(id));
     }
 
-    //ToDo: Create getAllPostsById in the user controller
+    @GetMapping("/{postId}/comments")
```

Comment:
Comment related methods should be inside commentsController, so getOnePost for example can be renamed back to getOne. The same applies for all other methods in this class